src/components/Alert/styles.module.scss

```diff
@@ -12,7 +12,7 @@
   background-color: #fa8f00;
   z-index: 9;
 
-  @media screen and (max-width: 800px) {
+  @media screen and (width <= 800px) {
```

Comment:
This syntax is not supported in chrome before 103(included).

Reply:
Only styled-components are not processed by postcss, which means they are not automatically downgraded to lower versions.

However, scss and css files with high-version syntax can be downgraded and made compatible after compilation.

For example:

<img src="https://github.com/Magickbase/ckb-explorer-frontend/assets/9160743/7a955d95-72a9-41f9-935c-18c31459eca2" width="49%" />

<img src="https://github.com/Magickbase/ckb-explorer-frontend/assets/9160743/a60c18a0-57f1-41c0-9d4e-0599a27e62bd" width="49%" />

Reply:
Cool